pkgs/development/python-modules/fasm/default.nix

```
@@ -0,0 +1,18 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "fasm";
+  version = "0.0.2.post100";
```

```suggestion
  version = "0.0.2.post100";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```

The chance is very small that somebody will test the package on Python releases which are EOL.